package.json

```diff
@@ -105,6 +106,7 @@
     "stylelint-processor-styled-components": "1.10.0",
     "supertest": "6.2.4",
     "ts-jest": "29.0.3",
+    "typescript": "4.8.4",
```

Comment:
I'm not sure we should target TS 4.8.4 here since we're using 4.6 in the rest of the features. Maybe we should update when everything else will be updated. What do you think?

Reply:
You're right, I just copied the version we were using in data-transfer. I've updated both to 4.6.2.